packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -1,3 +1,4 @@
+import isEqual from 'lodash/isEqual'
```

Comment:
we should settle on this convention vs. what we do elsewhere https://github.com/AudiusProject/audius-client/blob/6488d30511a49d0c379bfdc2651f09c6d775d189/packages/mobile/src/hooks/selectors.ts#L2

Reply:
ok will do - i used the other way to avoid importing the full library (since idt we have tree shaking? cc @dylanjeffers) but since we use `import { x } from 'lodash'` in a bunch of places it wouldn't make a difference on bundle size anyway

Reply:
i think import { x } from lo dash does tree shake. but actually what i'm saying is i like yours better :)

Reply:
Personally I like import {} since we have a babel plugin that modifies them into import blah from lodash/blah

Reply:
sgtm!